README.md

```diff
@@ -114,7 +126,7 @@ You can deploy this project using:
 
 - GitHub: [@newtonOmbese](https://github.com/mohammadnewton)
 - Twitter: [@NewtonOmbese](https://twitter.com/mohanewto200)
-- LinkedIn: [@newtonOmbese](https://www.linkedin.com/in/newton-ombese-570862210/)
+- LinkedIn: [@newtonOmbese](https://www.linkedin.com/in/mohammad-newton-ombese-570862210/)
```

Comment:
- [ ] according to the requirements The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).